### PR TITLE
full release of file restore

### PR DESCRIPTION
**Pull request by snopoke**
@benrudolph @czue 

cc @kaapstorm

### CODE REVIEW

File: corehq/ex-submodules/casexml/apps/phone/restore.py [hunk 1]
```diff
@@ -405,7 +352,7 @@ def create_sync_log(self):
 
     @property
     def restore_class(self):
-        return get_restore_class(self.domain, self.user)
+        return FileRestoreResponse
```
Comment (dannyroberts) [inline corehq/ex-submodules/casexml/apps/phone/restore.py]: Could even change this to be

`restore_class = FileRestoreResponse`

for extra points!

**Comment by dannyroberts** [comment]
Not a big deal, but do we want to have a standard way that random toggles get released? I noticed we're going from "50% released" to "100% released and old feature deleted", whereas there's some sort of principal of migrations in my mind that you want to release something in one step, watch nothing break, and then delete the old code path. I think the point is to minimize any possibility of added conflict in the case of a rollback.

That said, that's a little more overhead, so if you think you've reasoned with it and are pretty sure it's not worth it, this approach seems fine.

File: corehq/ex-submodules/casexml/apps/phone/restore.py (hunk 1 above)
Comment (snopoke) [inline corehq/ex-submodules/casexml/apps/phone/restore.py]: Updated

**Comment by snopoke** [comment]
@dannyroberts good point about the release path: https://github.com/dimagi/commcare-hq/pull/6953

**Comment by dannyroberts** [comment]
Sweet. (Merging this other expectedly caused a trivial conflict on this branch btw)

**Comment by benrudolph** [comment]
this looks good. thanks for doing this

**Comment by benrudolph** [comment]
is it possible to simplify some of the logic in `CachedResponse` now? haven't looked too closely

**Comment by snopoke** [comment]
@benrudolph simplified CachedResponse: https://github.com/dimagi/commcare-hq/commit/358afec02703f82fa0d8f7aad3370d54cf5b5534

**Comment by benrudolph** [comment]
:white_check_mark: from me. not merging cause of label. feel free to self merge

**Comment by snopoke** [comment]
Will wait a few more days to make sure there are no issues.

**Comment by benrudolph** [comment]
:+1:

**Comment by dannyroberts** [comment]
(not mergeable btw)

**Comment by dannyroberts** [comment]
oops lol i already said that

**Comment by snopoke** [comment]
@dannyroberts rebased master, also remove the 'hold off on merge' label

**Pull request merged**